mobile/src/main/java/org/hawkular/client/android/service/TriggerService.java

```diff
@@ -17,25 +17,43 @@
 package org.hawkular.client.android.service;
 
 import java.util.List;
+import java.util.Map;
 
+import org.hawkular.client.android.backend.model.Feed;
+import org.hawkular.client.android.backend.model.FullTrigger;
+import org.hawkular.client.android.backend.model.InventoryResponseBody;
+import org.hawkular.client.android.backend.model.Resource;
 import org.hawkular.client.android.backend.model.Trigger;
 
 import retrofit2.Call;
 import retrofit2.http.Body;
 import retrofit2.http.GET;
 import retrofit2.http.POST;
-import retrofit2.http.PUT;
-import retrofit2.http.Path;
+import retrofit2.http.QueryMap;
 
 public interface TriggerService {
 
     @GET("hawkular/alerts/triggers")
-    Call<List<Trigger>> get();
-
-    @PUT("hawkular/alerts/triggers/{id}")
-    Call<List<String>> updateTrigger(@Path("id") String id, @Body Trigger trigger);
-
-    @POST("hawkular/alerts/triggers/")
-    Call<List<String>> createTrigger(@Body Trigger trigger);
+    Call<List<Trigger>> get(
+            @QueryMap Map<String, String> parameters
+    );
+
+    @POST("/hawkular/alerts/triggers")
```

Comment:
@garg-anuj POST vs PUT is defined by the Rest-api of alerts. One has to look it up here: http://www.hawkular.org/docs/rest/rest-alerts.html#__triggers_triggers_definitions_handling